x-pack/plugins/security/server/plugin.ts

```diff
@@ -245,10 +255,25 @@ export class SecurityPlugin
     const kibanaIndexName = this.getKibanaIndexName();
 
     // A subset of `start` services we need during `setup`.
-    const startServicesPromise = core.getStartServices().then(([coreServices, depsServices]) => ({
-      elasticsearch: coreServices.elasticsearch,
-      features: depsServices.features,
-    }));
+    const startServicesPromise = core
+      .getStartServices()
+      .then(([coreServices, depsServices, startServices]) => ({
+        elasticsearch: coreServices.elasticsearch,
+        features: depsServices.features,
+        userProfiles: startServices.userProfiles,
+      }));
+
+    /**
+     * Once the UserProfileServiceStart is available, use it to start the SecurityPlugin > UserSettingService.
+     *
+     * Then the UserProfileSettingsClient is created with the SecurityPlugin > UserSettingServiceStart and set on
+     * the Core > UserSettingsServiceSetup
+     */
+    startServicesPromise.then(({ userProfiles }) => {
+      this.userSettingServiceStart = this.userSettingService.start(userProfiles);
+      const client = new UserProfileSettingsClient(this.userSettingServiceStart);
+      core.userSettings.setUserProfileSettings(client);
+    });
```

Comment:
Optional: In theory, you're not supposed to call `setup` methods from core once `start` has been initiated.  

So the correct approach would be to have `UserProfileSettingsClient` being instantiable without passing `userSettingServiceStart`, and then pass it during `start`.

But it could be done as a follow up

Reply:
Ah ok, I'd have to rethink this, but if you're ok with the current impl for now, with 8.8 FF so close, I'll prefer try to clean it up in a future issue.

Reply:
Yeah, fine with me